libraries/package-extractor/webpack.config.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const webpack = require('webpack');
+const { PreserveDynamicRequireWebpackPlugin } = require('@rushstack/webpack-preserve-dynamic-require-plugin');
+const PathConstants = require('./lib/PathConstants');
+
+module.exports = () => {
+  return {
+    context: __dirname,
+    mode: 'development', // So the output isn't minified
+    devtool: 'source-map',
+    entry: {
+      [PathConstants.createLinksScriptFilename]: {
+        import: `${__dirname}/lib-esnext/scripts/create-links.js`,
+        filename: `${PathConstants.scriptsFolderName}/[name]`
+      }
+    },
+    output: {
+      path: `${__dirname}/lib`,
```

Comment:
Maybe we should put the output into a `dist/scripts/*` path? That way the code that references it can point to the same location when using the `lib-esnext` or `lib` versions? Cause as it stands right now, the `lib-esnext` version would break because it paths to it using `${__dirname}/scripts/create-links.js`  (ln. 314 in PackageExtractor.ts)

Reply:
Yeah probably